src/app/sessions/reducers/index.ts

```diff
@@ -133,10 +133,14 @@ export const getFullSessionPropositions = createSelector(getSessionsPropositions
         return finalSessions;
     });
 
-export const getSessionById = (id: string) => createSelector(getFullSessions, (svm: SessionViewModel[]) => {
+export const getSessionViewModelById = (id: string) => createSelector(getFullSessions, (svm: SessionViewModel[]) => {
     return svm.find(s => s.id === id);
 });
 
+export const getSessionById = (id: string) => createSelector(getAllSessions, (sessions: Session[]) => {
+    return sessions.find(s => s.id === id);
```

Comment:
The 'find' method might cause us performance issues considering the amounts of sessions we have. Some of the selector returns sessions as a hashmap: <id, Session> which would be much faster